CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# How to contribute
```

Comment:
This appears to be leftover from the template this was pulled from, can it be removed?